flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/CommandLineWrapper.java

```diff
@@ -45,6 +46,11 @@ public WGetBuilder targetDir(Path dir) {
             return this;
         }
 
+        public WGetBuilder timeoutSecs(long secs) {
```

Comment:
This should accept a Duration and internally convert it to whatever unit wget expects